src/CLI.hs

```diff
@@ -93,7 +95,8 @@ data OptsRewrite = OptsRewrite
     maxDepth :: Integer,
     maxCycles :: Integer,
     targetFile :: Maybe FilePath,
-    inputFile :: Maybe FilePath
+    inputFile :: Maybe FilePath,
+    stepsDir :: Maybe FilePath
```

Comment:
@yarkivaev let's swap `inputFile` and `stepsDir` since first we should have all the options and `inputFile` as argument at the end